kernel/src/hil/screen.rs

```diff
@@ -90,6 +154,14 @@ impl ScreenPixelFormat {
     }
 }
 
+/// Returned for combined operations
+pub enum UpdateType {
+    /// All settings have been applied
+    Complete,
+    /// Not all settings have been applied
+    Partial,
+}
+
```

Comment:
I am not sure that I fully understand the purpose of this. Can you provide an example?

Reply:
I addressed this in another comment.